tests/cross/src/test/scala/tests/pc/CompletionSnippetSuite.scala

```diff
@@ -230,9 +225,8 @@ class CompletionSnippetSuite extends BaseCompletionSuite {
            |""".stripMargin,
       "3" ->
         // scala 3 type completions not implemented, so no way to distinguish if we are at a type position
```

Comment:
You can remove the comment above and the other places that we now properly support type completions.